tests/NPSP.py

```diff
@@ -307,6 +311,35 @@ def fill_address_form(self, **kwargs):
             else:
                 locator = locator+"input"
                 self.selenium.get_webelement(locator).send_keys(value)
+                
+    def fill_bge_form(self, **kwargs):
+        """Validates if the affiliated contacts have the added contact details enter Y for positive case and N for negative case""" 
+        for label, value in kwargs.items():
+            if label=="Batch Description":
+                locator= npsp_lex_locators['bge']['field-text'].format(label,value)  
+                self.salesforce._populate_field(locator, value)              
+
+            else:
+                locator= npsp_lex_locators['bge']['field-input'].format(label,value)
+                self.salesforce._populate_field(locator, value)
+
+    
+    def select_bge_date_picker(self,title): 
+        locator= npsp_lex_locators['bge']['field-input'].format(title)
+        self.selenium.set_focus_to_element(locator)
+        self.selenium.get_webelement(locator).click()     
+    
+    def pick_bge_date(self, value):
```

Comment:
@boakley it's used in bge_custom_batch.robot

Reply:
so i have added all these keywords as it would be easy to understand when reading test case but I removed all those and add click element with locator keyword which gets the locator and clicks